src/main/kotlin/com/bridgecrew/utils/CheckovUtils.kt

```diff
@@ -19,7 +19,15 @@ class CheckovUtils {
     companion object {
         private val LOG = logger<CheckovUtils>()
         fun isCustomPolicy(result: BaseCheckovResult): Boolean {
-            return (result.category == Category.IAC || result.category == Category.SECRETS) && !result.id.startsWith("CKV")
+            return isCustomPolicy(result.category, result.id)
+        }
+
+        fun isCustomPolicy(category: Category, id: String): Boolean {
+            return (category == Category.IAC || category == Category.SECRETS) && !id.startsWith("CKV")
```

Comment:
why do we need this new method?

Reply:
I use it in ResultsCacheService.kt for before converting Checkov result to BaseCheckovResult for checking if the result needs to be ignored, so we're saving the conversion commands + time